openedx/core/djangoapps/config_model_utils/utils.py

```diff
@@ -21,4 +22,8 @@ def is_in_holdback(user):
         except ExperimentData.DoesNotExist:
             pass
 
+    if enrollment is not None:
+        if FBEEnrollmentExclusion.objects.filter(enrollment=enrollment).exists():
```

Comment:
Huh. I think you should be able to use `enrollment.something` to check for the exclusion in a way you can `select_related` for, but I'm not sure exactly how. Maybe just `enrollment.fbeenrollmentexclusion`, and catch the `DoesNotExist`?